vuepress/docs/next/tutorials/compose/content-tutorial.md

```diff
@@ -1,108 +1,89 @@
 
-    -   **Categories.** Select the category(s) and select + to add a classification to the 
-        content to use for searching and sorting.  This field is not mandatory.
+    *  `Content Attributes`: Define each required `Attributes` field and any optional fields for the `Content Type` of this item. Only the `Date` and `Title` are required.
 
-    -   **Content Attributes.** Define each required attributes field and any optional fields
-        defined by the content types used for this Content. This field is mandatory.
+    * `Content Info - History`: This displays all the saved iterations for the item where you can view details or revert to a previous version. 
 
-![image](./img/Content3.png)
+![Add Content Information](./img/content-addinfo.png)
 
-Save the new content by selecting:
+## Saving and Approving Content 
+1. For new or edited content, determine which `Status` it belongs to:  
+`Status`:
+ * `Draft`: The content is in the development stage and not ready for approval or publication. The `Save` or `Save and Continue` buttons establish this status for all content.
 
--   **Save**: The content will be saved in a draft version.
+ * `Ready`: The content is ready for review and has not yet been approved nor published.  Items with this status can be saved and approved later or published directly.
 
--   **Save and Continue**: The content will be saved and the editable form will be displayed.
+2. Saving options:
 
--   **Save and Approve**: The content will be saved, approved and published.
+-   **Save**: The content will be saved in a draft version.
 
-## Finding Content
+-   **Save and Continue**: The content will be saved as a draft version and the editable form will be displayed.
 
-You can list specific Content by using the filters:
+-   **Save and Approve**: The content will be saved, approved, and ready for use in an application.
 
--   **All**: All created Content is displayed (Draft, To Approve and
-    Approved/Published)
 
--   **Pending Changes**: All created Content that was saved by only clicking on the
-    **Save** or **Save and Continue** button. The content is not
-    approved and therefore is not ready to be published.
+## Finding Content
 
--   **Ready for Approval**: All created Content that is in **Ready** status,
-    and saved by clicking on the **Save** or **Save and Continue** button.
-    The content is ready for review but has not yet been approved and
-    therefore has not been published.
+Using the `Advanced Filters` feature, you can search for Content by name, type, category, group and status. \
+![Content Filters](./img/content-filters.png)
 
--   **Published**: All created Content that has been saved and approved
-    by clicking on the **Save and Approve** button. This content has been
-    published.
+In the resulting table, contents are listed with these parameters: `Name`, `Author`
```

Comment:
contents -> `Contents`; not talking about general content or you'd say "content is"